backend/app/core/security.py

```
@@ -0,0 +1,34 @@
+from passlib.context import CryptContext
+from datetime import datetime, timedelta, timezone
+from fastapi import HTTPException, status
+from jose import JWTError, jwt
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+SECRET_KEY = "CHANGE_ME_TO_RANDOM_SECRET_KEY"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)